src/scss/components/_verification-badge.scss

```diff
@@ -3,23 +3,12 @@
   background-color: map-get($site-palette, 'site-white');
   border: units(1px) solid map-get($site-palette, 'site-green');
   box-sizing: border-box;
-  display: inline-block;
+  display: inline-flex;
+  align-items: center;
   padding: units(1) units(2);
   white-space: nowrap;
 
   img {
     margin-right: units(1);
   }
-
-  span {
-    vertical-align: text-top;
-  }
-}
-
-@media #{$theme-site-margins-breakpoint} {
```

Comment:
`$theme-site-margins-breakpoint` comes from USWDS and is equal to the string value `"desktop"` ([source](https://github.com/uswds/uswds/blob/e1ed21783b089a2831504971b3426fbea1b5b752/src/stylesheets/settings/_settings-spacing.scss#L90)). `@media desktop {` is not a valid media query. This would have needed to use [the `at-media` mixin](https://github.com/uswds/uswds/blob/develop/src/stylesheets/core/mixins/_at-media.scss) to do anything. Based on the styles, I'm assuming this is meant to address some of the IdP-specific styling of verification badges as right-aligned on the account page at desktop viewports. Since it's project-specific, I don't think it's appropriate to keep.